package.json

```diff
@@ -18,8 +18,8 @@
   },
   "productName": "vite-reactts-electron-starter",
   "scripts": {
-    "dev": "concurrently \"yarn dev:vite\" \" yarn dev:electron\"",
-    "dev:vite": "vite",
+    "dev": "concurrently \"npm run dev:vite\" \" npm run dev:electron\"",
+    "dev:vite": "npm run build:vite && vite",
```

Comment:
why do you build before starting dev server ?

Reply:
Hmm, never mind... I added it because it was present in for:
```json
"dev:electron": "npm run build:electron && electron .",
```
but it does not seem to be necessary in both cases